erplibre_website_snippets_cards/views/snippets.xml

```diff
@@ -0,0 +1,122 @@
+            <div class="card-extend__img">
+              <img class="o_image_fluid img img-fluid" src="/web/image/website.s_picture_default_image" alt="Product illustration"/>
+            </div>
+            <div class="card-extend__text bg-100">
+              <div class="card-extend__title">
+                <img class="img img-fluid" src="/erplibre_website_snippets_cards/static/src/img/cards_extend.png" alt="icon"/>
+                <h4 class="o_h4">Extend on Click</h4>
+              </div>
+              <p class="o_p">Duplicate Me. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </template>
+
+  <template id="s_cards_flip" name="Flippable Cards">
+    <section class="o_cards_flip pt16 pb16">
+      <div class="container">
+        <div class="row">
+          <div class="o_card_flip col-lg-4 col-md-6 col-sm-12">
+            <div class="card-flip__inner">
+              <div class="card-flip__front bg-200">
+                <h5>Make me flippable</h5>
+                <div class="o_avatar">
+                  <img class="img img-fluid" src="/erplibre_website_snippets_cards/static/src/img/portrait.png" alt="Avatar"/>
+                </div>
+                <h4>Duplicate Me</h4>
+                <div class="card-flip__icons">
+                  <a href="https://www.linkedin.com/">
+                    <i class="fa fa-linkedin-square"/>
+                  </a>
+                  <a href="https://github.com//">
+                    <i class="fa fa-github-square"/>
+                  </a>
+                  <a href="mailto: #">
+                    <i class="fa fa-envelope-o"/>
+                  </a>
+                </div>
+              </div>
+              <div class="card-flip__back bg-200">
+                <div>
+                  <h4>Duplicate Me</h4>
+                  <h5>Full Stack Developer</h5>
+                </div>
+                <p class="text-center">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+                <div class="card-flip__icons">
+                  <a href="https://www.linkedin.com/">
+                    <i class="fa fa-linkedin-square"/>
+                  </a>
+                  <a href="https://github.com/">
+                    <i class="fa fa-github"/>
+                  </a>
+                  <a href="mailto: #">
+                    <i class="fa fa-envelope-o"/>
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </template>
+
+  <!-- Add snippets to menu -->
+  <template id="add_snippet_structure_to_menu" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_structure']/div[@class='o_panel_body']" position="inside">
```

Comment:
Error-prone use of @Class in view add_snippet_structure_to_menu (): use the hasclass(*classes) function to filter elements by their classes

Reply:
updated to `div[hasclass('o_panel_body')]`